railties/test/commands/devcontainer_test.rb

```diff
@@ -16,7 +16,7 @@ class Rails::Command::DevcontainerTest < ActiveSupport::TestCase
     assert_match "app_name: app_template", output
     assert_match "database: sqlite3", output
     assert_match "active_storage: true", output
-    assert_match "redis: true", output
```

Comment:
It is a little hard to write a test for the non-solid case here, because we don't actually require the generated app's gems but just a hard coded list (which I added the solid gems to [here](https://github.com/rails/rails/pull/52934/files#diff-59df0083f17e516bde4b2bbb9b904b082d10a25ead45734e79edd47fd2819201)). So I would have to undefine them to test the non-solid option. Or, I could require those gems in this test case, but it's a bit odd since they are defaults.